lib/Email/Valid.pm

```diff
@@ -157,6 +158,15 @@ sub _net_dns_query {
     }
   }
 
+  # Chceck for A record for $host
+  my @a_rrs = Net::DNS::rr($Resolver, $host, 'A');
```

Comment:
The original MX and Host checks only looked for A records.

I suppose assing in AAAA would be a future proof check as well for IPV4.